### PR TITLE
feat: toaster component updated

### PR DESCRIPTION
**Pull request by PrashanthreddyAinala**
image

### CODE REVIEW

File: styles/toast.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@import url(./colors.css);
+
+.toast {
+  color: white;
+  padding: 1rem;
+  border-radius: 0.5rem;
+  width: 18rem;
```
Comment (sanket4120) [inline styles/toast.css:7]: You can use max-width here so that it will not break on smaller screens.

**Review by sanket4120 — APPROVED** [review]
Rest looks good to me

**Comment by jyothishp994** [comment]
Code looks good to merge

**Review by jyothishp994 — COMMENTED** [review]
Clean code, Indented and made use of Variable. That is great.

**Pull request merged**